azure-pipelines.yml

```diff
@@ -51,26 +51,27 @@ jobs:
     displayName: 'Build Docker Container'
 
   - script: |
-      if [[ $NAME == UNIT ]]; 
+      if [[ $NAME == UNIT ]];
       then docker run -t  --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest bash -c \
       "NUMBA_DISABLE_JIT=1 pytest -v arviz/tests/ --junitxml=junit/test-results.xml --cov=arviz/ --cov-report=xml --cov-report=html";
         fi
     displayName: 'Test ArviZ'
 
   - script: |
-      if [[ $NAME == LINT ]]; 
+      if [[ $NAME == LINT ]];
       then docker run -t arviz:latest bash -c "./scripts/lint.sh";
         fi
-        
+
+      if [ $? -ne 0 ]; then
+        exit 1
+        fi
```

Comment:
@canyon289 I think moving it here will make both lint jobs act the same. At least here they are doing the same. Does it also make sense to you? I will move it to a different PR after some more experiments.